src/NServiceBus.Gateway.AcceptanceTests/MyRequest.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace NServiceBus.Gateway.AcceptanceTests
```

Comment:
while keeping this message in the namespace for XML reasons, couldn't we still place it in the same file as the test (and also add a comment about why)?